container-images/tcib/base/os/nova-base/nova-compute/nova-compute.yaml

```diff
@@ -27,6 +26,4 @@ tcib_packages:
   - swtpm-tools
   - targetcli
   - xfsprogs
-  x86_64:
```

Comment:
You have to remove the reference in line 2 as well.

Reply:
ah yes i did not se that conditional install, will do.